crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -64,6 +64,15 @@ def upgrade!
             message: msg
           }
         end
+      rescue StandardError => e
```

Comment:
Here you are not caching usual exceptions from the library, so if I start the step in the incorrect order, it ends in this block with `StandardError` which is wrong